astropy/coordinates/representation/geodetic.py

```diff
@@ -62,6 +80,16 @@ def __init_subclass__(cls, **kwargs):
             raise AttributeError(
                 f"{cls.__name__} requires '_ellipsoid' or '_equatorial_radius' and '_flattening'."
             )
+        if cls._positive_longitude not in ["east", "west"]:
+            raise ValueError(
+                f"Invalid argument '{cls._positive_longitude}' for '_positive_logitude' "
+                "attribute: valid argument are 'east' or 'west'."
+            )
+        if cls._wrap_angle not in [180, 360]:
```

Comment:
I don't think this test is necessary; if people want to have a system with (-90, 270), why should we stop them?